lib/identity_doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -111,14 +111,14 @@ def create_response_info
           {
             ConversationId: conversation_id,
```

Comment:
If this method / hash return is now a common interface, any thoughts on what we might want to do with these extra vendor-specific properties? Are we content to keep them as-is? Or at least name them in a consistent format like `conversation_id` ? Or bucket them under some vendor-specific property value like a hash e.g. `extra_vendor_details` ?

Reply:
I went back and forth about it then forgot to come to any conclusion. I'll probably just fix the naming now. Those extra properties are just there for logs anyway.